src/caselawclient/types.py

```diff
@@ -1,4 +1,11 @@
-from typing import NamedTuple
+from dataclasses import dataclass, field
+from typing import List, NamedTuple
```

Comment:
I don't think you need to import the `List` here, you can just use the native `list`.